src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/gerritnotifier/GerritNotifier.java

```diff
@@ -84,7 +84,7 @@ public void buildStarted(Run build, TaskListener taskListener,
                         parameterExpander.getBuildStartedCommand(build, taskListener, (ChangeBasedEvent)event, stats);
                 if (command != null) {
                     if (!command.isEmpty()) {
-                        logger.info("Notifying BuildStarted to gerrit: {}", command);
+                        logger.info("Notifying BuildStarted to gerrit: {}", truncateCommandForLog(command));
```

Comment:
As an alternative we could log without the command on info and then log the command on debug (or lower).

Reply:
Yea, this shouldn't be info level at all I think

Reply:
Let's go that route, in general gerrit trigger is a very chatty plugin, changing this to debug would improve the situation somewhat :)

Reply:
Btw. why are we logging to the logger and not to the listener which ends up in the build log?

Reply:
I'm not sure I can remember correctly that far back. But I think it started as a debug log to monitor that the plugin was working correctly. then someone might have wanted it as an info. Or it was an info to begin with because changing logger configs are hard and then found the utility of it good to have and left it there. And we didn't start with listening to build started, that was added a bit later and I guess was copied from `onCompleted` where we didn't dare logging if the logger was closed or something.
_TL DR:_ No good reason at all :)

Reply:
Thanks for the explanation :)